packages/vkui/src/components/Alert/Alert.tsx

```diff
@@ -49,6 +50,7 @@ export interface AlertActionInterface
 
 export interface AlertProps
   extends Omit<React.HTMLAttributes<HTMLElement>, 'title'>,
+    Omit<UseFocusTrapProps, 'onClose' | 'autoFocus'>,
```

Comment:
```suggestion
    Pick<UseFocusTrapProps, 'restoreFocus'>,
```
предлагаю пока ограничиться `restoreFocus`

`mount`, `disabled` и `timeout` выглядят сомнительно, исключить пока не будем из `restProps`, будем по ситуации решать, либо префикс `focus`, либо свойство `FocusProps`